test/drenv/subctl.py

```diff
@@ -27,11 +27,20 @@ def deploy_broker(context, globalnet=False, broker_info=None, version=None, log=
         shutil.move(BROKER_INFO, broker_info)
 
 
-def join(broker_info, context, clusterid, cable_driver=None, version=None, log=print):
+def join(
+    broker_info,
+    context,
+    clusterid,
+    cable_driver=None,
+    version=None,
+    check_broker_certificate=True,
+    log=print,
+):
     """
     Run subctl join ... logging progress messages.
     """
     args = ["join", broker_info, "--context", context, "--clusterid", clusterid]
+    args.append(f"--check-broker-certificate={check_broker_certificate}")
```

Comment:
This will add:

    --check-broker-certificate=True

Which may not work since it should use "true|false".

Since by default subctl checks certs, we can do this:

    if not check_broker_certificates:
        args.append("--check-broker-certificate=false")

Reply:
I included this change in https://github.com/RamenDR/ramen/pull/1536

https://github.com/RamenDR/ramen/pull/1536/commits/f7bb75d9d3bd7cd16825bd823a14a7a53e75ea38